### PR TITLE
fix typo in README.md

### PR DESCRIPTION
**Pull request by clavichord93**
Change shadowsocks-libev to shadowsocksr-libev.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Then I rewrite the GUI code by swift.
 - SSR features!
 - Ability to check update from GitHub.
 - White domain list & white IP list
-- Use ss-local from shadowsocks-libev 2.5.6
```
Comment (qinyuhang) [inline README.md]: The process is still named ss-local, so the is no need for this.

**Comment by qinyuhang** [comment]
The psd file is backup for menu icon. DO NOT TRY REMOVE IT.

**Comment by clavichord93** [comment]
In the previous pull request, I duplicate the psd files so I just remove the duplicated ones. So it is okay to do so.

**Comment by qinyuhang** [comment]
sure this is good

**Pull request merged**